backends/arm/arm_backend.py

```diff
@@ -128,7 +128,7 @@ def ethosu_compile_spec(
         self.compiler_flags.append("--output-format=raw")
         self.compiler_flags.append("--debug-force-regor")
 
-        base_tosa_version = "TOSA-1.0+INT"
+        base_tosa_version = "TOSA-1.0+INT+int16"
```

Comment:
We have been using `ts.DType.INT16` in a few places, how did that work without this flag?